godot-core/Cargo.toml

```diff
@@ -27,6 +27,7 @@ godot-ffi = { path = "../godot-ffi" }
 # See https://docs.rs/glam/latest/glam/index.html#feature-gates
 glam = { version = "0.23", features = ["debug-glam-assert"] }
 serde = { version = "1", features = ["derive"], optional = true }
+godot-cell = { path = "../godot-cell" }
```

Comment:
> Out of curiosity, a separate crate was mostly motivated by being able to run miri + tests without interfering with Godot symbols? (..)

Yeah

> A good part is also that we can have a small, watertight boundary (no `unsafe` as you mentioned, but also no `pub(crate)` stuff leaking).

This too
 
> Do you plan to eventually move other parts into here (e.g. from `godot-core` or `godot-ffi`) -- in case they are godot-rust agnostic?

I didnt really plan on moving more stuff into this crate. Otherwise i would have named it differently.

Reply:
Makes sense!